src/test/regress/sql/multi_alter_table_add_constraints_without_name.sql

```diff
@@ -0,0 +1,227 @@
+--
+-- MULTI_ALTER_TABLE_ADD_CONSTRAINTS_WITHOUT_NAME
+--
+-- Test checks whether constraints of distributed tables can be adjusted using
+-- the ALTER TABLE ... ADD without specifying a name.
+
+SET citus.shard_count TO 32;
```

Comment:
If there is no specific reason for 32 shards, I think we don't need this line here and we can go with the default (which is 4 shards in the tests)

Reply:
Removed.